pkg/controllers/dynakube/activegate/consts/consts.go

```diff
@@ -28,6 +28,7 @@ const (
 	EnvDtHttpPort        = "DT_HTTP_PORT"
 
 	AnnotationActiveGateConfigurationHash = api.InternalFlagPrefix + "activegate-configuration-hash"
+	AnnotationActiveGateTokenHash         = api.InternalFlagPrefix + "activegate-token-hash"
```

Comment:
```suggestion
	AnnotationActiveGateTenantTokenHash         = api.InternalFlagPrefix + "activegate-tenant-token-hash"
```
I agree with @andriisoldatenko